.vscode/settings.json

```diff
@@ -0,0 +1,18 @@
+{
+  "editor.formatOnSave": true,
```

Comment:
This one is from my local config. I can remove it if you have different choices here. 😄

Reply:
Yup, I have my own vscode config and the main reason I "ignored" this PR is that I was annoyed that it contained its own config, sorry :sweat_smile: 

We can rename it to `settings.example.json`, and have this in our `.gitignore`:
```
.vscode
!.vscode/settings.example.json
```

and then in `CONTRIBUTING.md` mention that we have that example vscode config.

Reply:
https://github.com/microsoft/vscode/issues/40233

Yes, it looks like VS Code supports this a bit badly.